server/routes/prisonIncentiveLevels.test.ts

```diff
@@ -229,6 +231,19 @@ describe('Prison incentive level management', () => {
         .expect(404)
     })
 
+    it('should 404 if level is required globally (POST)', () => {
```

Comment:
Not too bothered about this on the UI side but I wonder if a `400 Bad Request` would be more appropriate in this case?

The level (Standard) is there but it can't be deactivated because it's required globally.

Just a thought :)

Reply:
yeah ok. particularly given there's already another error condition that returns a 400.

i'm a little wary of the more esoteric status codes for content returned to a _browser_… i have a vague recollection that they used to show their own error pages instead. anyway, this is a rare edge case that should never be seen by users.

Reply:
done! 26fc4ada68ad421a8b5e25a3587a23a0ba683ce5